src/game/common/rts/playerlist.h

```diff
@@ -18,46 +18,71 @@
 #include "snapshot.h"
 #include "subsysteminterface.h"
 
+class Team;
+
+constexpr int32_t MAX_PLAYER_COUNT = 16;
```

Comment:
should be a in a more global header, its gonna be used a lot

Reply:
Any suggestions as to which one?

Reply:
Suggested gametypes.h